sdk/iotcentral/Microsoft.Azure.Management.IotCentral/src/Generated/Models/App.cs

```diff
@@ -49,7 +49,7 @@ public App()
         /// application. Optional; if not specified, defaults to a blank
         /// blueprint and allows the application to be defined from
         /// scratch.</param>
-        public App(string location, AppSkuInfo sku, string id = default(string), string name = default(string), string type = default(string), IDictionary<string, string> tags = default(IDictionary<string, string>), string applicationId = default(string), string displayName = default(string), string subdomain = default(string), string template = default(string))
+        public App(string location, CustomAppSkuInfo sku, string id = default(string), string name = default(string), string type = default(string), IDictionary<string, string> tags = default(IDictionary<string, string>), string applicationId = default(string), string displayName = default(string), string subdomain = default(string), string template = default(string))
```

Comment:
Everything in `Generated/` folder will be completely replaced by next generation. So never touch anything in it

Reply:
Should I make the name same so it will automatically use my customization part of code?